packages/block-library/src/breadcrumbs/index.php

```diff
@@ -435,10 +435,12 @@ function block_core_breadcrumbs_get_archive_breadcrumbs() {
 			$post_type = reset( $post_type );
 		}
 		$post_type_object = get_post_type_object( $post_type );
+		$title            = apply_filters( 'post_type_archive_title', $post_type_object->labels->name, $post_type );
+
 		if ( $post_type_object ) {
 			// Add post type (current if not paginated, link if paginated).
 			$breadcrumb_items[] = block_core_breadcrumbs_create_item(
-				$post_type_object->labels->archives,
+				$title ?? $post_type_object->labels->archives,
```

Comment:
Maybe it's better to use or here (`$title || $post_type_object->labels->archives`)?  Not sure, but it feels some consumers of `post_type_archive_title` could return an empty string? 🤔

Reply:
I see what you mean, changed in https://github.com/WordPress/gutenberg/pull/73966/commits/3c8bc4ea7c841d3b85da556f13edb5189425e43d 👍

Reply:
What I found out with my suggestion and that change is the PHP's `||` operator returns a boolean and doesn't evaluate the value like JS. So we should do: `$title? $title : $fallback_label`.